tools/tsp-client/src/npm.ts

```diff
@@ -20,8 +20,12 @@ export async function createPackageJson(rootPath: string, deps: Set<string>): Pr
 }
 
 export async function npmCIDependencies(workingDir: string): Promise<void> {
+  const args = ["ci"];
+  if (process.env['TSPCLIENT_UNSAFE_FORCE'] === "force") {
```

Comment:
Open to feedback on the naming, I wanted it to be something that doesnt encourage a lot of usage